### PR TITLE
Support secure gRPC channel for AWS

### PR DESCRIPTION
**Pull request by DragonDriver**
Signed-off-by: dragondriver <jiquan.long@zilliz.com>
issue: #881

### CODE REVIEW

**Comment by DragonDriver** [comment]
You can just establish a secure connection via:
```python
connections.connect(host, port, secure=True)
```
In old style API:
```python
client = Milvus(host, port, secure=True)
```

File: pymilvus/client/grpc_handler.py [hunk 1]
```diff
@@ -100,13 +101,24 @@ def close(self):
...
+            if not self._secure:
+                self._channel = grpc.insecure_channel(
+                    self._uri,
+                    options=[(cygrpc.ChannelArgKey.max_send_message_length, -1),
+                             (cygrpc.ChannelArgKey.max_receive_message_length, -1),
+                             ('grpc.enable_retries', 1),
+                             ('grpc.keepalive_time_ms', 55000)]
+                )
+            else:
+                creds = grpc.ssl_channel_credentials(root_certificates=None, private_key=None, certificate_chain=None)
```
Comment (XuanYang-cn) [inline pymilvus/client/grpc_handler.py:113]: Is it valid that we don't provide root certificates and private_key?

**Comment by XuanYang-cn** [comment]
@DragonDriver Mostly lgtm, could you please change the commit message to something like `Support gRPC secured channel for AWS`

This PR seems a partial impl for #881 for AWS only.

**Pull request merged**